test-cases/longevity/ics_space_amplification_test.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 1500
+
+round_robin: true
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c4.2xlarge'
```

Comment:
A note, on why using bigger loaders might help, so people won't think it's by mistake or something like that (and to justify the pricing in a way)